02-04-domain-validations/internal/platform/server/handler/courses/create_test.go

```diff
@@ -64,4 +64,26 @@ func TestHandler_Create(t *testing.T) {
 
 		assert.Equal(t, http.StatusCreated, res.StatusCode)
 	})
+
+	t.Run("given an invalid course name in request it returns 400", func(t *testing.T) {
+		createCourseReq := createRequest{
+			ID:       "8a1c5cdc-ba57-445a-994d-aa412d23723f",
+			Name:     "too short",
+			Duration: "10 months",
+		}
+
+		b, err := json.Marshal(createCourseReq)
+		require.NoError(t, err)
+
+		req, err := http.NewRequest(http.MethodPost, "/courses", bytes.NewBuffer(b))
+		require.NoError(t, err)
+
+		rec := httptest.NewRecorder()
+		r.ServeHTTP(rec, req)
+
+		res := rec.Result()
+		defer res.Body.Close()
+
+		assert.Equal(t, http.StatusBadRequest, res.StatusCode)
```

Comment:
If I'm not wrong, you're returning a `411 Length Required` when the course name is too short. So, if that's what you're testing here, I think you need to update this assertion and the subtest name as well. Thanks!

Reply:
You are right!!